templates/contest/list.html

```diff
@@ -257,7 +257,7 @@ <h4>{{ _('Upcoming Contests') }}</h4>
         <br>
 
         {% if past_contests %}
-            <h4>{{ _('Past Contests') }}</h4>
+            <h4 id="past-contests">{{ _('Past Contests') }}</h4>
```

Comment:
This shouldn't be here.

Reply:
This is in the code currently: https://github.com/DMOJ/online-judge/blob/master/templates/contest/list.html#L260.